jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/BuildImageStepTest.java

```diff
@@ -46,6 +47,8 @@
 
   @Mock private BuildConfiguration mockBuildConfiguration;
   @Mock private BuildLogger mockBuildLogger;
+  @Mock private PullBaseImageStep mockPullBaseImageStep;
+  @Mock private Image<Layer> mockBaseImage;
```

Comment:
We could prob just have a real `Image` since there's nothing in Image that needs to be mocked.